GVFS/GVFS.Upgrader/GVFS.Upgrader.csproj

```diff
@@ -57,7 +57,9 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="App.config" />
+    <None Include="App.config">
+      <SubType>Designer</SubType>

```

Comment:
I don't know the purpose of this change. Why is it here?

Reply:
Ya... [looks like](https://github.com/Microsoft/msbuild/issues/2876) the build tools added it.  Manually removed it.